### PR TITLE
Bugfix for 'Registry' entity

### PR DESCRIPTION
**Pull request by oshtaier**

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -2823,6 +2823,18 @@ def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
         return {u'registry': super(Registry, self).update_payload(fields)}
 
+    def create_missing(self):
+        """Give the ``url`` instance attribute a value if it is unset.
+
+        By default, :meth:`nailgun.entity_fields.URLField.gen_value` does not
```
Comment (Ichimonji10) [inline nailgun/entities.py]: I really need to fix that method. :frowning:

**Comment by Ichimonji10** [comment]
APT

File: nailgun/entities.py (hunk 1 above)
Comment (Ichimonji10) [inline nailgun/entities.py]: See #189.

**Comment by Ichimonji10** [comment]
Bump.

This is blocking #2716.

**Comment by abalakh** [comment]
ACK

**Comment by elyezer** [comment]
:+1:

**Pull request merged**